daos/story-dao.ts

```diff
@@ -27,14 +29,17 @@ export default class StoryDao implements StoryDaoI {
 
   private constructor() {}
 
+
   /**
    * Inserts story instance into the database
    * @param {string} uid User's primary key
+   * @param {string} viewers Primary keys of users who can view the story, if vid is empty,
+   * story is available for public
    * @param {Story} story Instance to be inserted into the database
    * @returns Promise To be notified when story is inserted into the database
    */
-  createStory = async (uid: string, story: Story): Promise<Story> => {
-    return StoryModel.create({...story, postedBy: uid});
+  createStory = async (uid: string, viewers: string[], story: Story): Promise<Story> => {
```

Comment:
This list `viewers` will be present inside `Story` object right? Can we access it from there?

Reply:
yes! changed that part of passing a viewer as a separate parameter.